pyserini/prebuilt_index_info.py

```diff
@@ -4017,6 +4017,34 @@
 TF_INDEX_INFO = {**TF_INDEX_INFO_CURRENT, **TF_INDEX_INFO_DEPRECATED}
 
 IMPACT_INDEX_INFO_CURRENT = {
+    "msmarco-v1-passage-slim-bm25": {
```

Comment:
hrm... we already have `msmarco-v1-passage-slim`, so this is going to cause _a lot_ of confusion...
https://github.com/castorini/pyserini/blob/master/pyserini/prebuilt_index_info.py#L207

Reply:
yeah, I noticed it as well but I don't have any good ideas right now to resolve the name conflicts....